GroversAlgorithm/GroversAlgorithm.ipynb

```diff
@@ -288,14 +288,16 @@
     "\n",
     " * If the register is in any other basis state, multiply its phase by -1.\n",
     "\n",
-    "> This operation implements operator $2|0...0\\rangle\\langle0...0| - I$.\n",
+    "> This operation implements operator $2|0...0\\rangle\\langle0...0| - I$ $ = \\left(\\begin{matrix}1&0&…&0\\\\0&-1&…&0\\\\\\vdots&\\vdots&\\vdots&\\vdots\\\\0&0&0&-1\\end{matrix}\\right) $\n",
     "\n",
     "<br/>\n",
     "<details>\n",
     "  <summary>Need a hint? Click here </summary>\n",
     "    Note that quantum states are defined up to a global phase.\n",
     "    Thus the state obtained as a result of this operation is the same\n",
-    "    as the state obtained by flipping the sign of only the $|0...0\\rangle$ state.\n",
+    "    as the state obtained by flipping the sign of only the $|0...0\\rangle$ state and ignoring the global phase -1.<br>\n",
```

Comment:
Looking at my original phrasing with a fresh eye, I'd rephrase it a bit further: "Thus the state obtained as a result of this operation is *equivalent* to the state obtained by flipping the sign of only the $|0...0\\rangle$ basis state (those states differ by a global phase -1)"